docs/sidebars.js

```diff
@@ -126,63 +113,174 @@ module.exports = {
                 'guides/version-history',
                 'guides/how-to-create-multiple-projects',
                 'guides/customizing-the-appearance-of-your-project',
-                'guides/dbt-semantic-layer',
-                'guides/how-to-reset-your-password',
                 'guides/how-to-promote-content',
             ],
         },
         {
             type: 'category',
             label: 'References',
+            link: {
+                type: 'generated-index',
+                title: 'Lightdash Reference Docs',
+                description:
+                    'Reference docs are broken down into the categories below.',
+                slug: '/references',
+            },
             items: [
                 {
-                    type: 'doc',
-                    id: 'references/dimensions',
-                    label: 'Dimensions',
+                    type: 'category',
+                    label: 'CLI',
+                    link: {
+                        type: 'generated-index',
+                        title: 'CLI References',
+                        description:
+                            'Below are reference docs for all Lightdash CLI commands.',
+                        slug: '/references/cli',
+                    },
+                    items: [
+                        {
+                            type: 'doc',
+                            id: 'guides/cli/cli-authentication',
+                            label: 'Lightdash Login',
+                        },
+                        {
+                            type: 'doc',
+                            id: 'guides/cli/how-to-auto-generate-schema-files',
+                            label: 'Lightdash Generate',
+                        },
+                        {
+                            type: 'doc',
+                            id: 'guides/cli/how-to-use-lightdash-preview',
+                            label: 'Lightdash Generate',
+                        },
+                        {
+                            type: 'doc',
+                            id: 'guides/cli/how-to-use-lightdash-deploy',
+                            label: 'Lightdash Deploy',
+                        },
+                        {
+                            type: 'doc',
+                            id: 'guides/cli/how-to-compile-your-lightdash-project',
+                            label: 'Lightdash Compile',
+                        },
+                        {
+                            type: 'doc',
+                            id: 'guides/cli/how-to-use-lightdash-validate',
+                            label: 'Lightdash Validate',
+                        },        
+                    ]
                 },
                 {
-                    type: 'doc',
-                    id: 'references/metrics',
-                    label: 'Metrics',
+                    type: 'category',
+                    label: 'Analytics Engineering',
```

Comment:
I'm not sure about this label...like, I get the sentiment, but I thought that analytics engineering best principles was going to be under this reference doc. But actually, it's all about the semantic layer (is that analytics engineering?)

Maybe we could call it something like `developing your project` or `Lightdash development`?

Reply:
@TuringLovesDeathMetal  I agree and I also like `Lightdash Development` better so i'll make that update